src/tiered/tiered_handle.c

```diff
@@ -81,12 +93,16 @@ __tiered_init_tiers(WT_SESSION_IMPL *session, WT_TIERED *tiered, WT_CONFIG_ITEM
 static int
 __tiered_create_local(WT_SESSION_IMPL *session, WT_TIERED *tiered)
 {
+    WT_CONFIG cparser;
+    WT_CONFIG_ITEM ck, cv;
+    WT_DECL_ITEM(build);
     WT_DECL_RET;
     WT_TIERED_TIERS *this_tier;
     const char *cfg[4] = {NULL, NULL, NULL, NULL};
     const char *config, *name;
 
-    config = name = NULL;
+    config = NULL;
+    name = NULL;
```

Comment:
Not sure why you separated these as they're the exact same type. The change seems gratuitous. I don't have strong feelings either way.